<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1501</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矩阵计算器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1465</width>
      <height>1237</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
         </property>
         <property name="text">
          <string>查看矩阵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="matrixList">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="matrixView">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>600</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 32pt &quot;华文楷体&quot;;</string>
         </property>
         <property name="text">
          <string>矩 阵 计 算 器 v3.0 alpha</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="InputLine">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;Yu Gothic UI&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QPushButton" name="LRef">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>LRef</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_12">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="In">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>In</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="MRef">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>MRef</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Det">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>Det</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QPushButton" name="Num_2">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="Nor">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>Nor</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="C">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_34">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>CE</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="BackSpace">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>&lt;-</string>
             </property>
             <property name="shortcut">
              <string>Backspace</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="Clear">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>CLR</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QPushButton" name="Divide">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>/</string>
             </property>
             <property name="shortcut">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_32">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="left_bracket">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>(</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="Inv">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>Inv</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QPushButton" name="Multiply">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>*</string>
             </property>
             <property name="shortcut">
              <string>*</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="Num_9">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>9</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="A">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QPushButton" name="Subtract">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="pushButton_13">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_3">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="5" column="6">
            <widget class="QPushButton" name="Equal">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>=</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QPushButton" name="Num_0">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6">
            <widget class="QPushButton" name="Plus">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QPushButton" name="Num_6">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="Num_7">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="Num_8">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="Num_1">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QPushButton" name="Num_3">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="Num_4">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="Num_5">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QPushButton" name="right_bracket">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="D">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="B">
             <property name="styleSheet">
              <string notr="true">font: 25 10pt &quot;Microsoft YaHei UI Light&quot;;</string>
             </property>
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item row="5" column="5">
            <widget class="QPushButton" name="Num_Point">
             <property name="text">
              <string>.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTreeView" name="varList"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLineEdit" name="findName"/>
               </item>
               <item>
                <widget class="QPushButton" name="find">
                 <property name="styleSheet">
                  <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                 </property>
                 <property name="text">
                  <string>查找</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="addVariable">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>添加变量</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="deleteVariable">
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>删除变量</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="editVariable">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>修改变量</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="pushButton_25">
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>Nooooo</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="copy">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>复制</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="paste">
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
                   </property>
                   <property name="text">
                    <string>粘贴</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
         </property>
         <property name="text">
          <string>历史记录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="History">
         <property name="modelColumn">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="readFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAsFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="readFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="saveAsFile">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
